### PR TITLE
Descheduler modes

### PR DESCRIPTION
**Pull request by ingvagabund**
/hold

### CODE REVIEW

**Comment by ingvagabund** [comment]
Waiting for https://github.com/kubernetes-sigs/descheduler/pull/699

File: manifests/4.10/kube-descheduler-operator.crd.yaml [hunk 1]
```diff
@@ -47,6 +47,13 @@ spec:
                   description: managementState indicates whether and how the operator should manage the component
                   type: string
                   pattern: ^(Managed|Unmanaged|Force|Removed)$
+                mode:
+                  description: Modes configures the descheduler to either evict pods or to simulate the eviction
+                  type: string
+                  enum:
```
Comment (soltysh) [inline manifests/4.10/kube-descheduler-operator.crd.yaml:53]: We probably want `Predictive` to be the default, all the rest lgtm

Comment (ingvagabund) [reply manifests/4.10/kube-descheduler-operator.crd.yaml]: Done

**Review by ravisantoshgudimetla — APPROVED** [review]
/approve

You can remove the hold if you think this doesn't warrant units.

**Pull request merged**